example/App.js

```diff
@@ -62,6 +62,27 @@ export default function App() {
     console.log('trackOnce:', err);
   });
 
+  Radar.trackOnce({
```

Comment:
@gabrielalao Does a plain `trackOnce` with no parameters still work?  Basically, the call right above this one (line 59)

Reply:
Can you run that and paste the output here?

Reply:
@tjulien Yes, it's working.